group_vars/all.yml

```diff
@@ -21,15 +21,13 @@ network: calico
 
 # Change this to an appropriate interface, preferably a private network.
 # For example, on DigitalOcean, you would use eth1 as that is the default private network interface.
-network_interface: "eth1"
-calico_cni_opts: "interface={{ network_interface }}"
-flannel_cni_opts: "--iface={{ network_interface }}"
+network_interface: ""
```

Comment:
I think we should still have `eth1` here instead of leaving it blank or else it'd not work usually.

Reply:
Normally this only works in the lab. I tested it in production - there may be problems on different servers with different interface names. With an empty variable everything is ok.
In [Vagrantfile](https://github.com/kairen/kubeadm-ansible/pull/47/files#diff-d7b6c33578f505f2f083af0115257453R49) I left just in case

Reply:
@n0madic okay, I was wondering why you were including it. So by default flannel / calico uses the default network interface?

Reply:
I deployed kubernetes cluster many times in different environments and always the overlay network did not need additional settings. Calico itself is able to detect the necessary interfaces. Flannel also worked fine.